packages/client/window/src/windows/Popup.ts

```diff
@@ -52,10 +53,9 @@ function createPopupString(width: number, height: number): string {
                 ? window.top.outerWidth / 2 + window.top.screenX - width / 2
                 : window.outerWidth / 2 + window.screenX - width / 2;
         } catch (e) {
-            console.error(e);
+            console.log("Error could be due to cross-origin policy", e);
```

Comment:
is there a way to check for cross-origin before, instead of calculating this alternative in a catch block?

Reply:
What I mean is, I'd rather have explicit code branches because of a known possible error, than expecting an error to be thrown and reacting to it.